doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -710,6 +713,11 @@ Bluetooth libraries and services
 
   * Added the :c:func:`bt_scan_update_connect_if_match` function to update the autoconnect flag after a filter match.
 
+* :ref:`bt_le_adv_prov_readme`:
+
+  * Bluetooth device name provider by default provides data only when in pairing mode (:c:member:`bt_le_adv_prov_adv_state.pairing_mode`).
```

Comment:
I would prefer to use `provides` instead of `sends` here. The provider only passes the data to `bt_le_adv_prov` subsystem (then an application takes data from subsystem ands sends the data over radio).